pkg/handler/handler.go

```diff
@@ -26,6 +27,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	EmqxContainerName          = "emqx"
+	ReloaderContainerName      = "reloader"
+	ReloaderContainerImage     = "emqx/emqx-operator-reloader:0.0.1"
+	ManageContainersAnnotation = "apps.emqx.io/manage-containers"
```

Comment:
And this line should in this file, good job

Reply:
Both handler and emqx_handler refer to EmqxContainerName，so  EmqxContainerName is more suitable for handler